core/shared/src/main/scala/scalaz/zio/stream/Stream.scala

```diff
@@ -696,9 +696,9 @@ object Stream extends Serializable {
     }
 
   /**
-   * Constructs an infinite stream from a `Queue`.
+   * Constructs an infinite stream from a `Queue2`.
    */
-  final def fromQueue[A](queue: Queue[A]): Stream[Any, Nothing, A] =
+  final def fromQueue[R, E, A, B](queue: Queue2[R, E, A, B]): Stream[R, E, B] =
```

Comment:
Is it worth creating an `Stream[R1, E1, B1]`?

Reply:
Oh yeah, good catch.

Reply:
Actually no, unless I misunderstood, this is just turning a queue into a stream, so no widening/narrowing should occur.